controls/srg_gpos/SRG-OS-000250-GPOS-00093.yml

```diff
@@ -10,4 +10,7 @@ controls:
             - configure_ssh_crypto_policy
             - file_sshd_50_redhat_exists
             - sshd_include_crypto_policy
+            - sshd_use_strong_macs
```

Comment:
Do we really need this rule if `sshd_include_crypto_policy` already makes sure that sshd will use crypto-policies to define the MACs algorithms for example.

Reply:
with https://github.com/ComplianceAsCode/content/pull/13461, this rule will no longer be present in RHEL9 STIG